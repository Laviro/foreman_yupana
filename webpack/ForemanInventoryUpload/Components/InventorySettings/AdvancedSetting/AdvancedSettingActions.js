import { setSetting } from '../InventorySettingsActions';

export const handleToggle = (name, current) =>
  setSetting({
    setting: name,
    value: !current,
  });
