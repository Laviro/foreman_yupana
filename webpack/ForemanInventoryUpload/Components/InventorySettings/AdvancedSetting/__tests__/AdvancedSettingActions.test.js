import { testActionSnapshotWithFixtures } from '@theforeman/test';
import { handleToggle } from '../AdvancedSettingActions';

const fixtures = {
  'should handleToggle': () => handleToggle('autoUploadEnabled', false),
};

describe('AdvancedSetting actions', () =>
  testActionSnapshotWithFixtures(fixtures));
