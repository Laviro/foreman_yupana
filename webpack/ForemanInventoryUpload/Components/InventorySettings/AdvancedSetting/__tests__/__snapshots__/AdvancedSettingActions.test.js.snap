// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdvancedSetting actions should handleToggle 1`] = `
Array [
  Array [
    Object {
      "handleError": [Function],
      "key": "INVENTORY_SETTINGS",
      "params": Object {
        "setting": "autoUploadEnabled",
        "value": true,
      },
      "type": "post-some-type",
      "url": "/foreman_inventory_upload/set_setting",
    },
  ],
]
`;
